### PR TITLE
About page template

### PR DESCRIPTION
**Pull request by mrphemi**
Created a template for the `about the model` page and populated with dummy text while we await content from clients. Isolated some repeating patterns into components but generally there wasn't much repeating patterns. Might refactor when we get the real content.

### CODE REVIEW

**Review by alexandermendes — COMMENTED** [review]
Probably worth moving this file to `/components/Column/index.ts`, as that's now the structure for all components! The overall idea being that every component should be considered reusable from the outset, rather than having components specifically for the walkthroughs/live demo/about page

File: components/about/Column.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      <h2 className="heading-2">{title && title}</h2>
+
+      {children}
+
+      {button && <Button text={buttonText ?? ''} link={buttonLink ?? ''} />}
+    </div>
+  );
+};
+
+export default Column;
```
Comment (alexandermendes) [inline components/about/Column.tsx]: Nothing actually wrong with this, but here are some posts you might find interesting:

- https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad
- https://notes.webutvikling.org/default-exports/
- https://rajeshnaroth.medium.com/avoid-es6-default-exports-a24142978a7a

**Review by curiousways — CHANGES_REQUESTED** [review]
@mrphemi - can you make components out of all the patterns at this stage (rather than waiting for the live content). I would see these being something like:

- Page Intro (title and description - also note the homepage uses this pattern, and it looks different to what's currently on the about page)

- Inline Title (with props for including a background shape if desired)

- Text and Image (text left image right + text right image left options)

I'd see a combination of "major" layout components (ie Text and Image) and then smaller patterns that are used within layout components, such as Inline Title.

Thoughts from all welcome, thanks.

**Review by curiousways — CHANGES_REQUESTED** [review]
@mrphemi - why do we set the size of the `p` tag from an ID in the global stylesheet, as opposed to setting it in the component itself?

Can you also add a prop for switching left image / text position? (right/left + left/right), thanks

**Review by curiousways — APPROVED** [review]

**Pull request merged**